lib/motors/controller.cpp

```diff
@@ -4,6 +4,10 @@ namespace hyped::motors {
 
 Controller::Controller(core::ILogger &logger) : logger_(logger)
 {
+  // TODO: respective getters and setters to be added
+  float temperature;
+  float current;
+  ControllerState state;
```

Comment:
These should be data members declared under `private` in controller.hpp and should be initialised in the constructor initialiser list

Reply:
Also I was being stupid and forgot these are set from within controller so we don't need getters and setters